azureml-minimal/README.md

```diff
@@ -63,8 +72,11 @@ However, ZenML works seamlessly with the infrastructure provisioned through thes
 
     ```shell
     zenml stack import <STACK-NAME> <PATH-TO-THE-CREATED-STACK-CONFIG-YAML>
+    ```
+    Now, set the stack as an active stack

```

Comment:
I really like how you have separated these commands into two blocks. Makes it really easy to copy and paste!

Reply:
hah yes, that was the idea. Glad you noticed :)